app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -571,12 +556,21 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
         }
         return dialog;
     }
-    
+
     @Override
     public boolean isBackEnabled() {
-        return false;
+        return true;
     }
-    
+
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();

```

Comment:
@ctsims oh, you're right, it is identical. I've removed it in https://github.com/dimagi/commcare-odk/pull/531